README.md

```diff
@@ -278,6 +278,13 @@ More info on testing can be found [here](https://medium.com/@_rchaves_/testing-i
 
 To set up your elm project on Circle CI, copy over our ```circle.yml``` and ```circle-dependencies.sh``` and follow the instructions on [our Circle CI tutorial](https://github.com/dwyl/learn-circleci)
 
+## Flags
+Flags allow you to pass data from Javascript to Elm as part of the initial state of your application.
+
+See our working flags example [here](https://github.com/dwyl/learn-elm/tree/master/examples/elm-flags). Here we pass the URL from Javascript to Elm (via the script tags in our index.html).
+
+Run ```npm install```,```elm-package install``` and ```npm start``` to see the output
```

Comment:
after installing `elm-live` and saving it as a dependency in `package.json` the `npm start` script compiles the `Main.elm` and starts the server ... but it's a _blank page_:
![image](https://cloud.githubusercontent.com/assets/194400/24342290/f0917af0-12b7-11e7-9548-89ef6d161a93.png)

When I view-source on the page we see:
![image](https://cloud.githubusercontent.com/assets/194400/24342298/fc690424-12b7-11e7-9c7d-bf3f1f573bbb.png)

So the `elm.js` is not being loaded ...